homeassistant/components/dht/sensor.py

```diff
@@ -59,22 +70,22 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     SENSOR_TYPES[SENSOR_TEMPERATURE][1] = hass.config.units.temperature_unit
     available_sensors = {
-        "AM2302": Adafruit_DHT.AM2302,
-        "DHT11": Adafruit_DHT.DHT11,
-        "DHT22": Adafruit_DHT.DHT22,
+        "AM2302": adafruit_dht.DHT22,
+        "DHT11": adafruit_dht.DHT11,
+        "DHT22": adafruit_dht.DHT22,
     }
     sensor = available_sensors.get(config[CONF_SENSOR])
```

Comment:
`config[CONF_SENSOR]` is simply the pin number. But it seems that this is not working in the docker container:
https://github.com/Tob1as/rpi-sensors/blob/a339b7158eafa040a3072048f085a20cb021078f/dht_sensor.py#L49-L50
You might want to give this a try? I still have to set it up on my pi.